aten/src/ATen/native/cuda/Copy.cu

```diff
@@ -61,7 +61,7 @@ void copy_device_to_device(TensorIterator& iter, bool non_blocking) {
           copy_stream));
     }
   } else {
-    AT_DISPATCH_ALL_TYPES_AND3(kHalf, kBool, kBFloat16, iter.dtype(0), "copy_", [&] {
+    AT_DISPATCH_ALL_TYPES_AND_COMPLEX_AND3(kHalf, kBool, kBFloat16, iter.dtype(0), "copy_", [&] {
```

Comment:
this is fixed in this PR https://github.com/pytorch/pytorch/pull/35344

Reply:
removed